sds/Cargo.toml

```diff
@@ -40,7 +40,7 @@ serde_yaml = "0.9.34"
 lazy_static = "1.5.0"
 futures = "0.3.30"
 async-std = "1.13.0"
-reqwest = "0.12"
+reqwest = { version = "0.12", default-features = false, features = ["charset", "http2", "macos-system-configuration", "rustls-tls"] }
```

Comment:
You can probably just add `rustls-tls`, but this is fine as a quick fix until we can verify that is the same